ocs_ci/ocs/utils.py

```diff
@@ -940,6 +973,27 @@ def run_must_gather(log_dir_path, image, command=None, cluster_config=None):
     return mg_output
 
 
+def get_logs_ocp_mg_pods():
+    """
+    Get logs from OCP Must Gather pods
+
+    """
+    from ocs_ci.ocs.resources.pod import get_all_pods, get_pod_logs
+
+    namespaces = get_namespce_name_by_pattern(pattern="openshift-must-gather")
+    for namespace in namespaces:
+        pods_mg_ns = get_all_pods(namespace=namespace)
+        for pod_mg_ns in pods_mg_ns:
+            log.info(
+                f"*** ocp_mg_pod_name: {pod_mg_ns.name} ocp_mg_pod_namespace: {namespace} ***"
+            )
+            pod_logs = get_pod_logs(
+                pod_name=pod_mg_ns.name, namespace=namespace, all_containers=True
+            )
+            log.error(f"ocp mg pod logs:\n{pod_logs}")
```

Comment:
I would rather store the log file to the path where we have supposed to have the logs.
So you can store it in the: log_dir_path

So user if they will not find must gather, they can at least see the logs of MG to investigate what's going on with must gather.

Reply:
done